app/export_test.go

```diff
@@ -78,12 +79,7 @@ func TestExportUserChannels(t *testing.T) {
 	}
 	var preferences model.Preferences
 	preferences = append(preferences, preference)
-	channelMember := model.ChannelMember{
-		ChannelId: channel.Id,
-		UserId:    user.Id,
-	}
-	th.App.Srv.Store.Channel().SaveMember(&channelMember)
```

Comment:
`SaveMember` was silently failing, since the channel member relationship already existed from the test data.